### PR TITLE
Adding Performance Impact of Sidecar in the CNI benchmark Blog

### PR DESCRIPTION
**Pull request by amitmavgupta**
Adding Performance Impact of Sidecar in the CNI benchmark Blog as a step to enhance cross blog visibility.

Adding numbers from https://isovalent.com/blog/post/cilium-service-mesh/ to https://cilium.io/blog/2021/05/11/cni-benchmark/ as a first step.

### CODE REVIEW

**Comment by amitmavgupta** [comment]
@xmulligan @liquidat FYI

File: src/posts/2021-05-11-cni-benchmark/index.md [hunk 1]
```diff
@@ -478,6 +479,16 @@ The difference in latency is much more significant than the consumed CPU:
 
 <a name="env"></a>
 
+# Performance Impact of a Sidecar
+
+Besides avoiding the sheer amount of proxies that need to be run in a sidecar model, a significant advantage of a sidecar-free model is that we can avoid the requirement to run two proxies in between any connection. This is made possible by using Cilium at the network/node level to encrypt and authenticate or use the upcoming new mTLS model which separates the authentication from the transport. More details in this blog [Next-Generation Mutual Authentication with Cilium Service Mesh](https://isovalent.com/blog/post/2022-05-03-servicemesh-security/)
+
+![](images/performance_impact_sidecar.png)
+
+Reducing the number of proxies in the network path and choosing the type of Envoy filter has a significant impact on performance.  The above benchmark illustrates the latency cost of HTTP processing with a single Envoy proxy running the Cilium Envoy filter (brown) compared to a two-sidecar Envoy model running the Istio Envoy filter (blue). Yellow is the baseline latency with no proxy with no HTTP processing performed.
```
Comment (xmulligan) [inline src/posts/2021-05-11-cni-benchmark/index.md:488]:
```suggestion
Reducing the number of proxies in the network path and choosing the type of Envoy filter has a significant impact on performance.  The above benchmark illustrates the latency cost of HTTP processing with a single Envoy proxy running the Cilium Envoy filter (brown) compared to a two-sidecar Envoy model running the Istio Envoy filter (blue). Yellow is the baseline latency with no proxy and no HTTP processing performed.
```

File: src/posts/2021-05-11-cni-benchmark/index.md [hunk 2]
```diff
@@ -478,6 +479,16 @@ The difference in latency is much more significant than the consumed CPU:
 
 <a name="env"></a>
 
+# Performance Impact of a Sidecar
```
Comment (xmulligan) [inline src/posts/2021-05-11-cni-benchmark/index.md:482]:
```suggestion
# Performance Impact of Sidecars on a Service Mesh
```

File: src/posts/2021-05-11-cni-benchmark/index.md [hunk 3]
```diff
@@ -478,6 +479,16 @@ The difference in latency is much more significant than the consumed CPU:
...
+
+Besides avoiding the sheer amount of proxies that need to be run in a sidecar model, a significant advantage of a sidecar-free model is that we can avoid the requirement to run two proxies in between any connection. This is made possible by using Cilium at the network/node level to encrypt and authenticate or use the upcoming new mTLS model which separates the authentication from the transport. More details in this blog [Next-Generation Mutual Authentication with Cilium Service Mesh](https://isovalent.com/blog/post/2022-05-03-servicemesh-security/)
```
Comment (xmulligan) [inline src/posts/2021-05-11-cni-benchmark/index.md:484]:
```suggestion
Besides avoiding the sheer amount of proxies that need to be run in a sidecar service mesh model, a significant advantage of sidecarless service mesh is that we can avoid requiring running two proxies in between any connection. More details about Cilium's sidecarless service mesh can be found in this blog [Next-Generation Mutual Authentication with Cilium Service Mesh](https://isovalent.com/blog/post/2022-05-03-servicemesh-security/)
```

**Review by xmulligan — APPROVED** [review]

**Pull request merged**